.github/workflows/test_generator_backend.yml

```
@@ -0,0 +1,98 @@
+          else
+            echo "Getting push diff for backend files..."
+            git diff HEAD^..HEAD -- backend/ > backend_diff.patch
+            git diff HEAD^..HEAD --name-only -- backend/ > backend_diff_files.txt
+          fi
+          
+          echo "Diff files count: $(wc -l < backend_diff_files.txt)"
+          echo "Backend Diff Content:"
+          cat backend_diff.patch
+
+      - name: Setup Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.4)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)